README.rst

```diff
@@ -232,9 +232,10 @@ CoVE for OCDS and IATI can be run from the command line. To get a JSON file with
 
 .. code:: bash
 
-    ./ocds-cli --options file-name
+    ./ocds-cli --options files
 
-``file-name`` can be a JSON or an Excel file.
+``files`` can be a list of JSON or Excel files. Regular expressions are supported. The use of multiple files must be
```

Comment:
I think this is misleading. The command accepts a single file or a directory containing a number of files, so I would leave it as ```file-name``` adding that multiple files are also accepted (```e.g, dir-name/*.json```) passing the option ```--stream```